sbom/sbom_test.go

```diff
@@ -107,51 +86,47 @@ func testSBOM(t *testing.T, context spec.G, it spec.S) {
 				}
 			}
 
+			type license struct {
+				License struct {
+					Name string `json:"name"`
+				} `json:"license"`
+			}
+
+			type component struct {
+				Type     string    `json:"type"`
+				Name     string    `json:"name"`
+				Version  string    `json:"version"`
+				Licenses []license `json:"licenses"`
+				PURL     string    `json:"purl"`
+			}
+
 			var cdxOutput struct {
-				SerialNumber string `json:"serialNumber"`
-				Metadata     struct {
-					Timestamp string `json:"timestamp"`
+				BOMFormat   string `json:"bomFormat"`
+				SpecVersion string `json:"specVersion"`
+				Metadata    struct {
+					Component struct {
+						Type string `json:"type"`
+						Name string `json:"name"`
+					} `json:"component"`
 				} `json:"metadata"`
+				Components []component `json:"components"`
 			}
```

Comment:
Can we share the output types between this test and formatted reader since they're the same? Same question  goes for the other formats too.

Reply:
Done in the most recent commit.